src/geometry/ArborX_HyperBox.hpp

```diff
@@ -0,0 +1,118 @@
+/****************************************************************************
+ * Copyright (c) 2017-2022 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef ARBORX_HYPERBOX_HPP
+#define ARBORX_HYPERBOX_HPP
+
+#if (__cplusplus >= 201703L) && (KOKKOS_VERSION >= 30699)
+
+#include <ArborX_DetailsKokkosExtArithmeticTraits.hpp>
+#include <ArborX_DetailsKokkosExtMinMaxOperations.hpp>
+#include <ArborX_GeometryTraits.hpp>
+#include <ArborX_HyperPoint.hpp>
+
+#include <Kokkos_Macros.hpp>
+
+namespace ArborX
+{
+
+namespace Experimental
+{
```

Comment:
```suggestion
namespace ArborX::Experimental
{

```

Reply:
There is code outside of the `Experimental`, like specialization of `GeometryTraits`.

Reply:
So?  You can/should be doing `template <...> ArborX::GeometryTraits::dim<...> { ... }`

Reply:
Done.